posts/2020-05-15-six-years-of-rust.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "Six Years of Rust"
+author: The Rust Core Team
+---
+
+Today marks [Rust]'s sixth birthday since it went 1.0 in 2015. A lot has changed since then and especially over the past year, and Rust was no different. In 2020, there was no foundation yet, no const generics, and a lot organisations were still wondering whether Rust was production ready. 
+
+In the midst of the COVID-19 pandemic, hundreds of Rust's global distributed set of team members and volunteers shipped over 9 major releases of Rust and multiple patch releases. Today, "Rust in production" isn't a question, but a statement. The newly founded Rust foundation's has several members who value using Rust in production enough to help continue to support and contribute to its open development ecosystem.
```

Comment:
```suggestion
In the midst of the COVID-19 pandemic, hundreds of Rust's global distributed set of team members and volunteers shipped over 9 major releases of Rust and multiple patch releases. Today, "Rust in production" isn't a question, but a statement. The newly founded Rust foundation has several members who value using Rust in production enough to help continue to support and contribute to its open development ecosystem.
```

Another option would be something like "foundation's membership includes", with corresponding tweaks to the rest of the sentence.